src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/RuntimeEnvironment.cs

```diff
@@ -23,7 +23,10 @@ public static bool FromGlobalAccessCache(System.Reflection.Assembly a)
         }
         public static string GetRuntimeDirectory()
         {
-            return Path.GetDirectoryName(typeof(object).Assembly.Location) + Path.DirectorySeparatorChar;
```

Comment:
AppDomain.CurrentDomain.BaseDirectory actually gives you a real path on Project N and gives you what you would have gotten from typeof(object).Assembly.Location + Separator had Location actually worked.

This is, assuming, that this is even what we want GetRuntimeDirectory() to return in an AOT world...